app/views/foreman/smart_proxies/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if @smart_proxy.has_feature?(SmartProxy::PULP_NODE_FEATURE) -%>
+  <div ng-controller="CapsuleContentController">
+    <%= render :file => 'smart_proxies/show' %>
```

Comment:
hmm, reading this again, I'm not sure I'm following the logic, this means it will render foreman's show vs katello override ?